lib/at_cmd_parser/at_cmd_parser.c

```diff
@@ -61,6 +63,21 @@ static int at_parse_detect_type(const char **str, int index)
 		 * notification ID, (eg +CEREG:)
 		 */
 		set_new_state(NOTIFICATION);
+
+		if (!strncmp(tmpstr, "%HWVERSION", 10)) {
```

Comment:
Should these numbers be defines? ``AT_CMD_HWVERSION_LEN`` ```AT_CMD_SHORTSWVER_LEN`` etc?

Reply:
Fixed